src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -1158,15 +1118,15 @@ private string LocalizedLanguageName
                         ((ci = CultureInfo.GetUserDefaultCulture()) != null) &&
                         !CultureInfo.DefaultThreadCurrentUICulture!.Name.Equals(ci.Name))
                     {
-                        _sLocalizedLanguage = NativeLanguageName;
+                        sLocalizedLanguage = NativeLanguageName;
```

Comment:
Why do we need to set `sLocalizedLanguage = NativeLanguageName` here when it was set to that above?

Reply:
This is native language and not native name.

Reply:
I don't follow.

```C#
string sLocalizedLanguage = NativeLanguageName;

// some ifs, and no one touches `sLocalizedLanguage`
...

    sLocalizedLanguage = NativeLanguageName;
```

Why are we re-setting it to the same thing?

Reply:
Sorry, I thought this code in the DisplayName property. you are right we don't have to reset it here. I'll remove it.